ang/civicase/case/search/directives/search.directive.js

```diff
@@ -390,8 +390,12 @@
       } else {
         var status = [];
         if (filters.status_id && filters.status_id.length) {
-          _.each(filters.status_id, function (s) {
-            status.push(_.findWhere(caseStatuses, { name: s }).label);
+          _.each(filters.status_id, function (statusNameOrId) {
```

Comment:
Not sure of this change. The object is called `filters.status_id`, but it can be either name or id? 
Could you give me example why we need both. Why cant we just send the ID?

Reply:
This is a bit confusing, it seem at some point we were using the status name, but then switched to the value. The value is what we store when filtering by status. I changed the code so it only filters by the "status_id" (value)

The other curious thing is that the `filters.case_type_id` is a list of case type names, not their ids 🤷‍♂ but I left the name the same because it would take too much effort to change this name.